lib/output.js

```diff
@@ -1857,7 +1858,9 @@ function OutputStream(options) {
         if (output.option("quote_keys")) {
             output.print_string(key);
         } else if ("" + +key == key && key >= 0) {
-            output.print(make_num(key));
+            output.option("keep_numbers")
+                ? output.print(key)
+                : output.print(make_num(key));
```

Comment:
Let's use ternaries only when calling pure functions. This might not be the case across the whole codebase, but it should be what we're striding towards.

Reply:
Changed :)